qiskit_machine_learning/neural_networks/estimator_qnn.py

```diff
@@ -45,19 +46,38 @@ class EstimatorQNN(NeuralNetwork):
     their expectation value(s). Quite often, a combined quantum circuit is used. Such a circuit is
     built from two circuits: a feature map, it provides input parameters for the network, and an
     ansatz (weight parameters).
+    In this case a :class:`~qiskit_machine_learning.circuit.library.QNNCircuit` can be passed as
+    circuit to simplify the composition of a feature map and ansatz.
+    If a :class:`~qiskit_machine_learning.circuit.library.QNNCircuit` is passed as circuit, the
+    input and weight parameters are held as internal properties of the circuit and are therefore
+    omitted.
 
     Example:
 
     .. code-block::
 
         from qiskit import QuantumCircuit
         from qiskit.circuit.library import ZZFeatureMap, RealAmplitudes
+        from qiskit_machine_learning.circuit.library import QNNCircuit
 
         from qiskit_machine_learning.neural_networks import EstimatorQNN
 
         num_qubits = 2
+
+        # Using the QNNCircuit:
+        # Create a parameterized 2 qubit circuit composed of the default ZZFeatureMap feature map
+        # and RealAmplitudes ansatz with 3 repetitions.
```

Comment:
The 3 reps here is the default value that RealAmplitudes has right. So its `a default ZZFeatureMap feature map with a default RealAmplitudes ansatz` - the latter has 3 reps by default. Instead of explicitly configuring the number or reps below, as was originally done, if we want it to match it might be better to remove the explicit setting and let it just set to the same default. It could be noted that this is 3 reps here if thats important.

Reply:
Mentioning the default reps for RealAmplitudes (reps=3) is not of particular importance. It was only meant to give the reader a hint why the length of the weights array is 8 😄 . Since the default reps of ZZFeatureMap (reps=2) are also not mentioned, let's ignore it.